eng/pipelines/release-cli.yml

```diff
@@ -213,7 +213,7 @@ stages:
               targetPath: installer
 
           # CLI ref docs
-          - pwsh: New-Item -ItemType Directory -Path docs
+          - pwsh: New-Item -Force -ItemType Directory -Path docs
```

Comment:
from: https://github.com/Azure/azure-dev/pull/765

Reply:
This fix would alleviate the error message but we'd be uploading additional stuff from the `docs/` folder that's not specific to our docs work. 

The real fix here is to have the docs artifact work happen outside of the repo (or at least in a folder whose path/name doesn't collide with other work we're doing)